ansible/roles/ocp-workload-migration/tasks/pre_workload.yml

```diff
@@ -75,7 +75,7 @@
     
 - name: "Applying variables"
   replace:
-    path: "{{ repodir.path }}/controller.yml"
+    path: "{{ repodir.path }}/controller-4.yml"
```

Comment:
I only see a `controller.yml` in the master & rhte2019 branches of the repo you are cloning...are you sure you want to change this here to reference a file that doesn't currently exist?

Reply:
@stencell Thank you for pointing that out. But, we already have a PR which will eventually make that file available at that location.